ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e.yaml

```diff
@@ -448,7 +479,21 @@ objects:
           # create a new floating ip tagged with CLUSTER_NAME so it can be deleted later
           APPS_FIP=$(openstack floating ip create --description $CLUSTER_NAME $OPENSTACK_EXTERNAL_NETWORK --format value -c 'floating_ip_address' --port $INGRESS_PORT)
           # create A record for *.apps
-          curl -v $CI_DNS_IP:8080/add -d '{"cluster_name": "'$CLUSTER_NAME'", "apps_fip": "'$APPS_FIP'"}'
+          cat > /tmp/artifacts/installer/ingress-record.json <<EOF
+          {
+          "Comment": "Create the public OpenShift API record",
+          "Changes": [{
+              "Action": "UPSERT",
+              "ResourceRecordSet": {
+                "Name": "*.apps.$CLUSTER_NAME.$BASE_DOMAIN.",
+                "Type": "A",
+                "TTL": 300,
+                "ResourceRecords": [{"Value": "$APPS_FIP"}]
+                }
+          }]}
+          EOF
```

Comment:
You'll need to decrease the indentation for this line otherwise bash doesn't interpret the EOF marker.

Reply:
Thanks so much!